src/vs/workbench/services/backup/node/backupModelService.ts

```diff
@@ -0,0 +1,72 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Yeah that's covered in `backupService`, it could also potentially be removed if the change events are buffered in https://github.com/Microsoft/vscode/issues/14186